ui/apps/everest/.e2e/release/restore-new-cluster.e2e.ts

```diff
@@ -0,0 +1,494 @@
+// everest
```

Comment:
Please use the `scheduled-backup.e2e.ts` test and copy your steps to that test from `Restore to a new database cluster [${restoredClusterName}]`.